htdocs/API/2.0/index.php

```diff
@@ -10,6 +10,21 @@
 
 require_once 'configuratie.include.php';
 
+/**
+ * Maak de API toegankelijk vanaf bepaalde externe domeinen.
+ */
+if (in_array($_SERVER['HTTP_ORIGIN'], explode(',', API_ORIGINS), true)) {
+	header('Access-Control-Allow-Origin: ' . $_SERVER['HTTP_ORIGIN']);
+	header('Access-Control-Max-Age: 1440');
+	header('Access-Control-Allow-Headers: Accept, Origin, Content-Type, X-Csr-Authorization');
+	header('Access-Control-Allow-Methods: PUT, GET, POST, DELETE, OPTIONS');
+
+	if ($_SERVER['REQUEST_METHOD'] === 'OPTIONS') {
+		http_response_code(204);
+		exit;
+	}
+}
```

Comment:
De naamgeving kan inderdaad beter naar mijn onbescheiden mening, want het is niets meer dan een [policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) waar externen (gezien vanuit ons domein) zich aan moeten houden.

Maargoed, waarom ik op deze PR reageer is om alleen maar heel duidelijk [aan te geven](https://github.com/csrdelft/csrdelft.nl/pull/293#discussion_r119740039) dat dit dus niets doet qua toegangscontrole _server-side_; en dat moet wel glashelder zijn voor met name nieuwe/toekomstige PubCie programmeurs.
Ik hoop niet dat ik onbedoeld ongewenste feedback geef. Je bent goed bezig @Manduro De vereniging is vast blij met je, ik in ieder geval wel, ga zo door!